pkg/webui/console/views/device-add/wizard/network-settings-form/index.js

```diff
@@ -180,6 +190,7 @@ const NetworkSettingsForm = props => {
             }
             component={Input.Generate}
             onGenerateValue={generate16BytesKey}
+            glossaryId={glossaryId.NETWORK_SESSION_KEY}
```

Comment:
What about `fNwkSIntKey` ? We have separate names and descriptions of the field but only 1 glossary entry? @benolayinka

Reply:
I think it's sufficient to make the distinction in the glossary entry itself. @benolayinka can you add a commit here to do that?

Reply:
https://github.com/TheThingsIndustries/lorawan-stack-docs/pull/6

Reply:
this reminds me, we're going to have to change the way docs are linked in the console when we move the repo

Reply:
> I think it's sufficient to make the distinction in the glossary entry itself. @benolayinka can you add a commit here to do that?

added the items but i didn't build so double check for typos 😬

Reply:
So how to proeed with this now, when the docs are removed?

Reply:
@kschiffer the default doc link is thethingsstack.io anyway so it shouldn't break
 https://github.com/TheThingsNetwork/lorawan-stack/blob/8648a86012b5e26cd5713d1ee566a423e62fd66c/cmd/internal/shared/console/config.go